make-it/EmMakefile

```diff
@@ -166,7 +166,10 @@ EM_EXPORTS = -s EXPORTED_FUNCTIONS="[\
     '_TypeRef_IsTimestamp',\
     '_TypeRef_IsComplex',\
     '_TypeRef_IsAnalogWaveform',\
-    '_JavaScriptInvoke_GetParameterType',\
+    '_TypeRef_IsJavascriptRefnum',\
+    '_JavaScriptInvoke_GetParameterTypeName',\
```

Comment:
JavaScriptInvoke_GetParameterTypeName is no longer needed correct? we should remove it.